lib/screens/v2/widget_instance/subway_status.ex

```diff
@@ -152,14 +164,22 @@ defmodule Screens.V2.WidgetInstance.SubwayStatus do
           green: serialize_green_line(grouped_alerts)
         }
 
-      length(routes_with_alerts) == 1 and get_total_alerts(grouped_alerts) == 2 ->
-        # Show both alerts in two rows
-        %{
-          blue: serialize_multiple_alert_rows_for_route(grouped_alerts, "Blue"),
-          orange: serialize_multiple_alert_rows_for_route(grouped_alerts, "Orange"),
-          red: serialize_multiple_alert_rows_for_route(grouped_alerts, "Red"),
-          green: serialize_multiple_alert_rows_for_route(grouped_alerts, "Green")
-        }
+      length(multi_alert_route_ids) == 1 ->
+        # Make sure the one multi-alert route has only two alerts.
+        multi_alert_route_id = List.first(multi_alert_route_ids)
+        alerts_for_multi_alert_route = Map.get(grouped_alerts, multi_alert_route_id)
+
+        if length(alerts_for_multi_alert_route) == 2 do
+          # Show both alerts in two rows
+          %{
+            blue: serialize_multiple_alert_rows_for_route(grouped_alerts, "Blue"),
+            orange: serialize_multiple_alert_rows_for_route(grouped_alerts, "Orange"),
+            red: serialize_multiple_alert_rows_for_route(grouped_alerts, "Red"),
+            green: serialize_green_line(grouped_alerts)
+          }
+        else
+          serialize_one_row_for_all_routes(grouped_alerts)
```

Comment:
No this is for LCD too. We can't show 3 alerts in any case. This conditional is basically if we have only one route with multiple alerts and that one route has only 2 alerts, show both of them in two rows. Otherwise, show one row that is either `Normal Service`, the one alert, or a summary of how many alerts there are.